src/components/CurrentChat/current-chat.module.css

```diff
@@ -153,7 +153,7 @@
   position: absolute;
   background: #fff;
 
-  top: 59px;
+  top: 60px;
   left: -321px;
   height: 800px;
   width: 370px;
```

Comment:
I do not recommend using absolute values for height and width, except for images (png, etc) and places where the width or height is not determined by the content. If there is content inside the block and there is a certain distance from the content to the edges, then you can set paddings. This applies to the entire project. In the project code, in addition to this, I saw many places where similar bones are used. This leads to bugs in adaptability and difficulties in further code support.